imsv-docs-astro/src/styles/tailwind.css

```diff
@@ -35,15 +35,16 @@
     color: inherit;
     text-decoration: inherit;
   }
-  h1,
+  /* This is breaking the standard markdown styling */
+  /* h1,
   h2,
   h3,
   h4,
   h5,
   h6 {
     font-size: inherit;
     font-weight: inherit;
-  }
+  } */
```

Comment:
could not see any negative impact from this, only positive (page headers actually became appropriately sized)

Reply:
Please move to separate PR.

Reply:
ok removed